roles/edpm_ovn/meta/argument_specs.yml

```diff
@@ -105,35 +81,18 @@ argument_specs:
         default: 8000
         description: ''
```

Comment:
So just for clarity, the arguments which are not in argument_specs, if still passed those will be noop or will get considered?

Reply:
If you pass them to the role it will produce error. Argspecs are something like function API, in this context.